charm_version

```diff
@@ -1 +1 @@
-1.0+69db8cf+69db8cf-dirty+69db8cf-dirty+69db8cf-dirty+69db8cf-dirty+d0f9b21
+1.0+91f8440-dirty+91f8440-dirty+91f8440-dirty+91f8440-dirty+91f8440-dirty+91f8440-dirty+91f8440-dirty
```

Comment:
Please set this to `1.0`

Reply:
Good point, thanks for bringing it up.
Do we want to keep it 1.0 permanently?

Reply:
Done